src/main.c

```diff
@@ -995,49 +995,49 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 
 	switch (evt->type) {
 	case CLOUD_EVT_CONNECTING:
-		LOG_INF("CLOUD_EVT_CONNECTING");
+		LOG_DBG("CLOUD_EVT_CONNECTING");
```

Comment:
I think its nice to have CLOUD events at log level INFO. This is useful information and nice to have printed by default by the application.